packages/frontend-shared/cypress.config.ts

```diff
@@ -11,8 +11,8 @@ export default defineConfig({
   reporter: '../../node_modules/cypress-multi-reporters/index.js',
   reporterOptions: {
     configFile: '../../mocha-reporter-config.json',
-    videoCompression: false, // turn off video compression for CI
```

Comment:
Surprised this wasn't a typescript error.

Reply:
same